src/contract/userop/index.ts

```diff
@@ -0,0 +1,40 @@
+import { VerifiedWallet } from "../../wallet";
+import { USEROP_DATATYPES, UseropResponse, UseropVerifiedContract } from "./contract";
+
+interface TransactionParams{
+    value:string
+}
+interface ClassParams {
+    apiKey:string, 
+    paymasterUrl:string,
+    wallet:VerifiedWallet
+    signer:VerifiedWallet
+    // options?:Options
+}
+class GasLessTransaction extends UseropVerifiedContract {
```

Comment:
@herberthk 

This GaslessTransaction class is not required at all, why is it required ?